### ISSUE TITLE
Error generating Mocks with @GenerateNiceMocks annotation

### ISSUE THREAD

**Issue by darshan-cygnet** [dart-lang/mockito#774]
Hello There,

I am trying to Mock a class with @GenerateNiceMocks, when I setup @GenerateNiceMocks and tried to build the generated files I encountered following problem.

    Invalid @GenerateMocks annotation: Mockito cannot generate a valid mock 
    class which implements 'CLASS TO MOCK' for the following reasons:

    The property accessor 'CLASS.VARIABLE=' features a private parameter type, '_ANOTHERCLASS', and cannot be stubbed. 
    Try generating this mock with a MockSpec with 'unsupportedMembers' or a dummy generator (see 
    https://pub.dev/documentation/mockito/latest/annotations/MockSpec-class.html).

Following is replica of my code:

    class CLASSTOMOCK {
        late final _ANOTHERCLASS classVariable;

        CLASSTOMOCK(){
            classVariable = _ANOTHERCLASS();
        }
    }

I have tried with all the options of Mocking like **@GenerateMocks**, **@GenerateNiceMocks** and creating a new class by extending **Mock** and implementing **CLASSTOMOCK**. But no luck.

I am also using **unsupportedMembers** option in **@GenerateNiceMocks** but it's not working properly.

Thank you in advance for helping me.

**Comment by srawlins** [comment]
Thanks for filing an issue. Please give a complete minimal reproduction for your case. Especially how you used unsupportedMembers. For example something like this test case: https://github.com/dart-lang/mockito/blob/master/test/builder/custom_mocks_test.dart#L489

**Comment by darshan-cygnet** [comment]
First of all Thank you @srawlins for the answer.
My issue has been resolved.

The actual issue was that I was using a library, and there was a private class in that Library. Now one class was having a variable of that Private class. And I was trying to mock that class. It was showing error. I have added the variable in the unsupportedMembers, but the initialization error was showing with the '=' sign.

Now, as per the link that you have shared, I have added another Symbol with '=' sign and it worked. So we can closing the issue.

Thank you so much once again.
